website/docs/litmus-installation-with-ingress.md

```diff
@@ -5,34 +5,66 @@ sidebar_label: Litmus with Ingress
 ---
 
 ---
+
 ### Install LitmusPortal with Ingress
 
 With Litmus-2.0.0-Beta3, LitmusPortal can be installed with ingress.
-In the following doc, we will use the Nginx ingress controller for ingress setup. 
-
+In the following doc, we will use the Nginx ingress controller for ingress setup.
 
 1. Install LitmusPortal in ClusterMode
+
 ```bash
 kubectl apply -f https://litmuschaos.github.io/litmus/2.0.0-Beta/litmus-2.0.0-Beta.yaml
 ```
 
-2. By default, the service type is NodePort. We have to patch it to ClusterIP
+2. By default, the service type is NodePort. We have to edit the service types to ClusterIP as given below
+
+```bash
+kubectl edit svc litmusportal-frontend-service -n litmus
+kubectl edit svc litmusportal-server-service -n litmus
+```
+
+3. Install Nginx Ingress Controller along with Kubernetes RBAC roles and bindings
 
 ```bash
-kubectl patch svc litmusportal-frontend -n litmus -p '{"spec": {"type": "ClusterIP"}}'
-kubectl patch svc litmusportal-server -n litmus -p '{"spec": {"type": "ClusterIP"}}'
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/mandatory.yaml
+```
+
+4. create the ingress-nginx ingress controller service as a load balancer service
+
+Sample service manifest
+
+```yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: ingress-nginx
+  namespace: ingress-nginx
+  labels:
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
+spec:
+  type: LoadBalancer
```

Comment:
Can we use CLusterIP? Because most of the on-prem cluster don't have LoadBalancer svc type